plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -270,7 +270,7 @@ def main():
 
             apps=dict(default=None, required=False),
             cache_table=dict(default=None, required=False, type='str'),
-            clear=dict(default=None, required=False, type='bool'),
+            clear=dict(default=False, required=False, type='bool'),
```

Comment:
All options in `specific_boolean_params` are only tested for being truthy, so `None` and `False` are teated the same way.

Reply:
Personally I think it is better to remove it, but other than that, using a default that is actually in the domain of values for the specified type does look better.